src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -1207,6 +1207,7 @@ INSERT INTO tab_categories VALUES('VascularEndotheliumVolume','Volume of vascula
 INSERT INTO tab_categories VALUES('ExpressionParameters','Expression parameters','Expression parameters','Individual',NULL);
 INSERT INTO tab_categories VALUES('DiseaseStates','Disease States','Disease state parameters','Individual',NULL);
 INSERT INTO tab_categories VALUES('Individual_AgeDependent','Individual age dependent','Individual age dependent','Individual','Individual');
+INSERT INTO tab_categories VALUES('OntogenyFactors','Ontogeny factors','Ontogeny factors','Individual','Individual');

```

Comment:
@Yuri05 New category and calculation method
Idea: OntogenyFactors CM is only used in the Expression Profile. Individual etc do not use it. That way, I am only generating the formula when required